src/UserService.Models.Db/DbUserGender.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace LT.DigitalOffice.UserService.Models.Db
+{
+  public class DbUserGender
+  {
+    public const string TableName = "UserGenders";

```

Comment:
UsersGenders

Reply:
Accordingly, do not forget to change it everywhere

Reply:
done